packages/react-dom-bindings/src/client/ReactDOMHostConfig.js

```diff
@@ -521,15 +552,15 @@ export function appendChildToContainer(
 export function insertBefore(
   parentInstance: Instance,
   child: Instance | TextInstance,
-  beforeChild: Instance | TextInstance | SuspenseInstance,
+  beforeChild: InstanceSibling,
```

Comment:
What is this new type needed for? It doesn't seem like you're exporting a new type so where is the input coming from?

Reply:
removed